src/app/components/user-profile/user-profile.component.ts

```diff
@@ -10,9 +9,8 @@ import {MyOption} from '../../model/MyOption';
 })
 export class UserProfileComponent implements OnInit {
   userProfileForm: FormGroup;
-  countries = Object.keys(CountriesEnum).map(key => ({value: CountriesEnum[key], label: key} as MyOption));
```

Comment:
You kinda rollback everything @edwinyachoui-ibm did so far, that doesn't seem right to me ... is there a specific reason for that ?